src/picterra/client.py

```diff
@@ -220,6 +220,26 @@ def upload_raster(
         self._wait_until_operation_completes(resp.json())
         return raster_id
 
+    def list_folder_detectors(self, folder_id: str):
+        """
+        List of detectors assigned to a given folder
+
+        Args:
+            folder_id (str): The id of the folder to obtain the detectors for
+
+        Returns:
+            A list of detector dictionaries
+
+        Example:
```

Comment:
leftover? I don't think we need it at this stage (internal use)

Reply:
I left this as a placeholder and forgot to fill it in, I've added an example just now which we should keep or will this mess with the docs generation?

Reply:
Ah no, let's keep it then, thx